lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -1823,7 +1845,6 @@ class MetricsEndpointAggregator(Object):
     derive from `MetricsEndpointAggregator` overriding the `_get_targets()`
     method, which is responsible for aggregating the unit name, host
     address ("hostname") and port of the scrape target.
-
```

Comment:
Docstring could use some editing.
- Unclear what MetricsEndpointAggregator adds on top of scrape-config/target
- An explanation of what's going on in terms of pure prom config is missing
- The instantiation example isn't too helpful

Reply:
Honestly, this is not relevant and not worth holding anything on.

`MetricsEndpointAggregator` is not even a little related to `scrape-config|target`. It's more like `LMAv1MetricsRelationInterfaceTransformer`.

What's going in terms of pure prom config is flat-out reverse engineering. Since nobody other than `cos-proxy` will ever use it (and it's never going to be supported anywhere else), skipping docs updates is a better use of time.